packages/spruce-next-helpers/src/skillskit/index.js

```diff
@@ -263,7 +263,15 @@ const skill = {
 
 	confirm: function() {
 		const confirm = {
-			show: (data: Object, onConfirm?: Function, onCancel?: Function) => {
+			show: ({
+				data,
```

Comment:
See comment below, but we usually mixin callbacks with other props, like this screen: https://www.dropbox.com/s/ruzcb29dq6scstl/Screenshot%202019-06-08%2012.02.27.png?dl=0

so i think we should drop `data` from here.

Reply:
updated